src/app/(mobile-ui)/home/page.tsx

```
@@ -0,0 +1,249 @@
+
+                                                            </div>
+                                                        )}
+                                                        
+                                                    </Card.Content>
+                                                )}
+
+                                            </Card>
+                                        </motion.div>
+                                    )
+                                })}
+                            </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)